tests/dbus/test_dbus_client.cpp

```diff
@@ -178,10 +181,13 @@ int main()
                         {
                             exit(-1);
                         }
-                        api->Attach("Test", 0x5678, extpanid, {}, {}, UINT32_MAX, [&api](ClientError aErr) {
```

Comment:
Keep the old test so we can test both versions of `Attach`?

Reply:
The test ran test with the same parameter twice before this change. I turned the 2nd attach into the using active dataset.